src/index.js

```diff
@@ -94,12 +94,17 @@ L.Control.SplitMap = L.Control.extend({
     if (!this._map) {
       return this
     }
-    if (this._leftLayer) {
+    if (this._leftLayer && this._leftLayer.getContainer) {
```

Comment:
I think the first part of the condition (`this._leftLayer && `) is not needed. Because in the `else` part you use it anyway. You can simply remove it, `this._leftLayer` and `this._rightLayer` are unlikely to be undefined when used in ipyleaflet.